stock-tracker/src/App.tsx

```diff
@@ -7,9 +7,10 @@ import { LatestNews } from "./features/latestNews";
 import { Overview } from "./features/overview";
 import { Footer } from "./features/footer";
 import "./assets/styles/App.css";
+import { AppState } from "./store/rootReducer";
 
-function App() {
-  const symbol = useSelector(state => state.search.symbol);
+const App = () => {
```

Comment:
Have you thought about/looked at strictly typing your components?

Currently const App is of type `( ) => JSX.Element`

You can strictly type this to `const App: FC = ( ) =>` 

React.FunctionComponent => React.FC => FC

Reply:
I do need to look more into this area and will get my components more strictly typed in a future PR.